docs/web-apps/automated-testing/playwright.md

```diff
@@ -65,6 +65,12 @@ You can run `saucectl` locally via Docker ([Installation Requirements](https://d
       <th>End of Life</th>
     </tr>
     <tbody>
+    <tr>
+      <td rowspan='2'>1.25.1</td>
+      <td><b>macOS:</b> 11.00, 12</td>
+      <td rowspan='2'>Chromium, Firefox, Webkit</td>
+      <td rowspan='2'>Sep 7, 2023</td>
+    </tr>
```

Comment:
Adds missing `tbody` tags.
```suggestion
    </tr>
    </tbody>
    <tbody>
```

Reply:
Thanks. Turns out I was missing a row as well as the tbody. Just pushed fixes.